woocommerce/woocommerce-gutenberg-products-block
# Add Checkout block sidebar

Part of #1304.
Part of #1486.

### Screenshots

![imatge](https://user-images.githubusercontent.com/3616980/76306278-e8fa8400-62c6-11ea-956c-9daa889005a6.png)

### How to test the changes in this Pull Request:

1. Create a post with a _Checkout_ block and verify the sidebar is shown with the correct values.
2. Verify you can add/remove coupons.
3. Verify shipping address is updated when it changes in the form.
4. Verify there are no regressions in _Cart_ block styles.
